### DISCUSSION TITLE
Wireguard + WGDashboard config files

### DISCUSSION THREAD

**Discussion by kuchar91** [tteck/Proxmox#419] [Q&A]
Hi, where can i find config files for created peers? There's no configs in /etc/wireguard/configs. Only clients.txt. In dashboard i have 3 configured peers. Also after typing pivpn list in lxc it says theres no configs.

**Comment by Koliera** [comment]
Had a look and also couldn't find them... Weird. In the end you can just scan them with the mobile app from the Dashboard and then export them to a zip file from the mobile app.
